docs/apps/02-features.mdx

```diff
@@ -0,0 +1,91 @@
+
+|                                           |
+| ----------------------------------------- |
+| ![ChainList](/screenshots/chain-list.png) |
+
+### The universal streaming engine
+
+You are in full control of the streaming model: linear, linear with cliffs, exponential, logarithmic or any other
+dynamic curve you can dream up.
+
+|                                      |
+| ------------------------------------ |
+| ![Gallery](/screenshots/gallery.png) |
+
+### Streamlined operations
+
+Create multiple streams at once quickly. Save time when creating streams for your employees, investors, or community
+members.
+
+|                                         |
+| --------------------------------------- |
+| ![Group](/screenshots/create-group.png) |
+
+### Support for every asset
+
+Stream with your favorite asset or token from day one!
+
+|                                                |                                                         |
+| ---------------------------------------------- | ------------------------------------------------------- |
+| ![Token List](/screenshots/tokenlist-home.png) | ![Token List Search](/screenshots/tokenlist-search.png) |
+
+### Detailed panels
+
+Explore the app and every available stream. Simulate future payouts, understand historical events or simply enjoy cool
+representations of a monetized passage of time (NFTs 🔥).
+
+|                                      |                                                                                                                       |
+| ------------------------------------ | --------------------------------------------------------------------------------------------------------------------- |
+| ![Details](/screenshots/details.png) | ![App](/screenshots/stream-profile.png) ![Simulate](/screenshots/simulation.png) ![Preview](/screenshots/preview.png) |
+
+### Be in control of your future
+
+Manage your streams 24/7 as you see fit. The app will guide you through every supported process and help you keep an eye
+on your payouts. Remember, you can always be both a sender and a recipient.
+
+|                                              |                                          |
+| -------------------------------------------- | ---------------------------------------- |
+| ![Withdraw](/screenshots/cover-withdraw.png) | ![Cancel](/screenshots/cover-cancel.png) |
+
+| Feature                | Recipient | Sender |
+| ---------------------- | :-------: | :----: |
+| Create Streams         |           |   ✅   |
+| Renounce Cancelability |           |   ✅   |
+| Cancel                 |    ✅     |   ✅   |
+| Cancel Multiple        |    ✅     |   ✅   |
+| Transfer               |    ✅     |        |
+| Withdraw               |    ✅     |        |
+| Allow                  |           |   ✅   |
+
+| Feature      | User |
+| ------------ | ---- |
+| Create Proxy | ✅   |
```

Comment:
I'll remove this because creating proxies is not a featured offered by Sablier - Sablier only needs the proxy to be set up (for various reasons).